doc/release-notes/9880-info-api-zip-limit-embargo.md

```diff
@@ -0,0 +1,5 @@
+Implemented the following new endpoints:
+
+- getZipDownloadLimit (/api/info/zipDownloadLimit): Get the configured zip file download limit. The response contains the long value of the limit in bytes.
+
+- getEmbargoEnabled (/api/info/embargoEnabled): Know if the Dataverse instance has been configured to allow embargoes.
```

Comment:
Something feels off to me...

We should probably make an active decision about two things:

- Should we continue putting "settings" in the path? 
- Should we match the name of the setting including the colon?

Another way to ask this is which of the following we want.

```
/api/info/zipDownloadLimit
/api/info/settings/zipDownloadLimit
/api/info/settings/:ZipDownloadLimit
```

For reference here is a sorted list of "info" endpoints as of this PR:

```
/api/info/apiTermsOfUse
/api/info/embargoEnabled
/api/info/server
/api/info/settings/:DatasetPublishPopupCustomText
/api/info/settings/incompleteMetadataViaApi
/api/info/version
/api/info/zipDownloadLimit
```

Reply:
It seems appropriate to me to add the setting subpath, and even include the colon if it is more understandable for the setting name, as long as the endpoint only reads the setting value, with no other operations around it.

For example, the `/admin/settings/{name}` endpoint returns the plain value of the setting if it exists, otherwise it returns a not found error.

On the other hand, the `info/zipDownloadLimit`endpoint does more than read the setting, since if the associated setting is not found it assigns a default value. So I find appropriate to exclude the "settings" endpoint subpath/naming here.

Same for the `info/embargoEnabled` endpoint, that after reading the `:MaxEmbargoDurationInMonths` setting, based on its possible values, checks whether the embargo functionality is enabled or not.

Reply:
@GPortas I wrote some docs to merge into this PR once we are happy with them:

- #9890